crates/kas-core/src/theme/text.rs

```diff
@@ -397,101 +396,62 @@ impl<T: FormattableText> Text<T> {
         Ok(self.unchecked_display())
     }
 
-    /// Configure text
-    ///
-    /// Text objects must be configured before use.
-    #[inline]
-    pub fn configure(&mut self) -> Result<(), InvalidFontId> {
-        // Validate default_font_id
-        let _ = fonts::library().first_face_for(self.font_id)?;
-
-        self.status = self.status.max(Status::Configured);
-        Ok(())
-    }
-
-    fn prepare_runs(&mut self) -> Result<(), NotReady> {
+    fn prepare_runs(&mut self) {
         match self.status {
-            Status::New => return Err(NotReady),
-            Status::Configured => self
+            Status::New => self
                 .display
-                .prepare_runs(&self.text, self.direction, self.font_id, self.dpem)
-                .map_err(|_| {
-                    debug_assert!(false, "font_id should be validated by configure");
-                    NotReady
-                })?,
+                .prepare_runs(&self.text, self.direction, self.font, self.dpem)
+                .expect("no suitable font found"),
```

Comment:
`prepare_runs` may now return `Err(NoFontMatch)`. I opted to handle this via panic here since it *shouldn't happen* (there should always be some fallback font/glyph, even if it's just the "tofu" box).